codex-rs/config.md

```diff
@@ -243,6 +243,17 @@ By default, `reasoning` is only set on requests to OpenAI models that are known
 model_supports_reasoning_summaries = true
 ```
 
+## include_plan_tool
+
+Controls whether to expose the experimental plan tool (named `update_plan`) to the model and include the corresponding guidance in the system prompt.
+
+When enabled, the model can call `update_plan` to keep an up-to-date, step-by-step plan for the task and Codex will render plan updates in the UI. When disabled, the tool is not advertised to the model and the “Plan updates” section is omitted from the prompt; any unsolicited `update_plan` calls will be treated as unsupported.
+
+```toml
+# Enable the experimental plan tool and prompt instructions
+include_plan_tool = true
```

Comment:
Can we prefix this with `experimental_` since I wouldn't be surprised if this goes away? Note we have `experimental_resume`.